tests/__snapshots__/schema.test.ts.snap

```diff
@@ -1,5 +1,28 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`enum can be filtered on 1`] = `
+"enum EA {
+  A
+  B
+  C
+}
+
+type MA {
+  fa: Int!
+  fb: EA!
+}
+
+input MAWhereUniqueInput {
+  fa: Int
```

Comment:
@Weakky 

1. The test and [the example](https://github.com/prisma/nexus-prisma/pull/385/files#diff-97a48ec5ce235535fdb2c30a627f12e7R210) are producing different results 🤔 
2. I think `Int` is wrong here. For example `fa: EA` would make a lot more sense? I have a feeling this `Int` comes from the backing prisma enum (which is an enum in sense of names over ints).